mautic/mautic
# Create configuration option for DateRange form type instead of hardcoded 30 days

![screen shot 2018-05-16 at 2 22 47 pm](https://user-images.githubusercontent.com/3016737/40136120-03ac318a-5915-11e8-96f2-0411b5e42ae4.png)


**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | NO
| New feature? | YES
| Automated tests included? | NO
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
Create a field in the system configuration form to set a default range for the DateRange Form that appears on Dashboard and other View pages. Options include: 

- current day
- last 24 hours
- last 48 hours
- last 72 hours
- 1 week
- 2 weeks
- 3 weeks
- 1 month
- 2 months
- 3 months

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. 
2. 

#### Steps to test this PR:
1. Open the system configuration at path: /s/config/edit
2. Select a value.
3. Clear cache
4. Verify date ranges on dashboard or campaign page: /s/campaigns/view/[some id]



#### List deprecations along with the new alternative:
1. 
2. 

#### List backwards compatibility breaks:
1. 
2.
